### PR TITLE
Enable 2023.1.0.dev20230728

### PR DESCRIPTION
**Pull request by artanokhov**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
     sha256: c3a443e26539f866969242e690cf0651ef629149741ee18732f954c734da6763  # [aarch64 or arm64]
     folder: src/plugins/intel_cpu/thirdparty/ComputeLibrary  # [aarch64 or arm64]
-  - url: https://github.com/openvinotoolkit/oneDNN/archive/f1080be59e8eaa053195cd06da833f6515e29a96.tar.gz
-    sha256: dfac2fc8a728ec8f5310e5144c3e4be547752bdcc45db1461249a73c5ff245a4
+  - url: https://github.com/openvinotoolkit/oneDNN/archive/5bf389f616d0b1551ce468f2712df492a7aab140.tar.gz
+    sha256: 31af79c5b25a01ae795ad03e0745d238f0459b6e79844c33f5cf8b549d005393
     folder: src/plugins/intel_cpu/thirdparty/onednn
+  - url: https://github.com/openvinotoolkit/mlas/archive/519abf79de5ee295cfe5bbed97037a2623616c80.tar.gz
+    sha256: ef681b16a0f8b325aa01ab94d252f4cbd2f4848d8a1f8a3f2bd12ee76b241238
+    folder: src/plugins/intel_cpu/thirdparty/mlas
```
Comment (ilya-lavrenov) [inline recipe/meta.yaml:24]: let's add `win64 or linux64`

Comment (artanokhov) [reply recipe/meta.yaml]: But it also failed on OSX

Comment (ilya-lavrenov) [reply recipe/meta.yaml]: Oh, right. 
This option is available only for x86_64 https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/CMakeLists.txt#L37, which implies arm64 as well.

Comment (artanokhov) [reply recipe/meta.yaml]: I see now it's for x86_64
https://github.com/openvinotoolkit/openvino/blob/5572fad7129b2ae859cd167aa3365f4caa2fb624/src/plugins/intel_cpu/CMakeLists.txt#L37

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
     - patches/004-gflags-samples.patch
   - url: https://github.com/ARM-software/ComputeLibrary/archive/refs/tags/v23.02.1.tar.gz  # [aarch64 or arm64]
     sha256: c3a443e26539f866969242e690cf0651ef629149741ee18732f954c734da6763  # [aarch64 or arm64]
     folder: src/plugins/intel_cpu/thirdparty/ComputeLibrary  # [aarch64 or arm64]
-  - url: https://github.com/openvinotoolkit/oneDNN/archive/f1080be59e8eaa053195cd06da833f6515e29a96.tar.gz
-    sha256: dfac2fc8a728ec8f5310e5144c3e4be547752bdcc45db1461249a73c5ff245a4
+  - url: https://github.com/openvinotoolkit/oneDNN/archive/5bf389f616d0b1551ce468f2712df492a7aab140.tar.gz
+    sha256: 31af79c5b25a01ae795ad03e0745d238f0459b6e79844c33f5cf8b549d005393
     folder: src/plugins/intel_cpu/thirdparty/onednn
+  - url: https://github.com/openvinotoolkit/mlas/archive/519abf79de5ee295cfe5bbed97037a2623616c80.tar.gz  # [not aarch64]
```
Comment (ilya-lavrenov) [inline recipe/meta.yaml:22]:
```suggestion
  - url: https://github.com/openvinotoolkit/mlas/archive/519abf79de5ee295cfe5bbed97037a2623616c80.tar.gz  # [not aarch64 and not arm64]
```
as for arm plugin

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**